src/fondant/component/image.py

```diff
@@ -0,0 +1,9 @@
+import typing as t
+from dataclasses import dataclass
+
+
+@dataclass
+class Image:
```

Comment:
This needs way more functionality to actually make the images with extra dependencies and the script available

Reply:
Yes but I guess we'll have shared functionality between runners that could be abstracted here